deployments/arbitrum.diamond.staging.json

```diff
@@ -206,8 +210,12 @@
         "Version": "1.0.0"
       },
       "0x00f2a92544347C59c6980ACb53fcF9E5408e9942": {
-        "Name": "",
-        "Version": ""
+        "Name": "NEARIntentsFacet",
```

Comment:
we have 4 NearIntentsFacets in our staging diamond now. Please remove the outdated ones.

Reply:
removed